### PR TITLE
Add Job Post Feature Completed

### PR DESCRIPTION
**Pull request by ManivannaBalaji**
- [x] Add job post feature completed

### CODE REVIEW

File: assets/js/AddJobPost.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+/**
+ * Function to add new job post.
+ */
+function addJob(){
+    event.preventDefault();
+    let jobTitle = document.getElementById('jobTitle').value;
```
Comment (csys-fresher-batch-2021) [inline assets/js/AddJobPost.js:6]: Use const.

File: views/recruiter/AddJobPost.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        <h5>Post a Job</h5>
+        <form onsubmit="addJob()">
+            <div class="form-group">
+                <label for="jobTitle">Job Posting Headline</label>
+                <input type="text" name="jobTitle" id="jobTitle" placeholder="Job Posting Headline" required>
+            </div>
+            <div class="form-group">
+                <label for="jobType">Employment Type</label>
+                <select name="jobType" id="jobType" required>
+                    <option value="Full">Full Time</option>
```
Comment (csys-fresher-batch-2021) [inline views/recruiter/AddJobPost.html:31]: <option disabled> -- SELECT -- </option>

File: views/recruiter/AddJobPost.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    </nav>
+    <header>
+        <h1 class="display-4 text-center">Recruit Desk</h1>
+    </header>
+    <main class="content">
+        <h5>Post a Job</h5>
+        <form onsubmit="addJob()">
+            <div class="form-group">
+                <label for="jobTitle">Job Posting Headline</label>
+                <input type="text" name="jobTitle" id="jobTitle" placeholder="Job Posting Headline" required>
```
Comment (csys-fresher-batch-2021) [inline views/recruiter/AddJobPost.html:26]: Add autofocus

File: views/recruiter/AddJobPost.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+                <label for="maxSalary">To</label>
+                <input type="number" name="maxSalary" id="maxSalary" required>
+            </div>
+            <div class="form-group">
+                <label for="location">Location</label>
+                <input type="text" name="location" id="location" placeholder="Location" required>
+            </div>
+            <div class="form-group">
+                <label for="noOfVacancy">No of Vacancies</label>
+                <input type="number" name="noOfVacancy" id="noOfVacancy" placeholder="No of Vacancy" required>
```
Comment (csys-fresher-batch-2021) [inline views/recruiter/AddJobPost.html:63]: Add min and max attribute

File: views/recruiter/AddJobPost.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+            <div class="form-group">
+                <label for="qualification">Educational Qualification</label>
+                <input type="text" name="qualification" id="qualification" placeholder="Qualification" required>
```
Comment (csys-fresher-batch-2021) [inline views/recruiter/AddJobPost.html:67]: Change text box to Select .

File: views/recruiter/AddJobPost.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+            <button class="btn btn-danger" id="postBtn">Post Job</button>
```
Comment (csys-fresher-batch-2021) [inline views/recruiter/AddJobPost.html:69]: Delete = danger class would be suitable.

**Review by ghost — APPROVED** [review]

**Pull request merged**